pebblo/app/storage/sqlite_db.py

```diff
@@ -0,0 +1,74 @@
+
+from sqlalchemy import create_engine, func
+from sqlalchemy.orm import sessionmaker
+
+from pebblo.log import get_logger
+
+from .database import Database
+
+logger = get_logger(__name__)
+
+
+class SQLiteClient(Database):
+    def __init__(self):
+        super().__init__()
+        # Create an engine that stores data in the local directory's my_database.db file.
+        self.engine = create_engine("sqlite:///pebblo.db", echo=True)
```

Comment:
same here

Reply:
I'll take care of this in the follow up PR.